mlflow/cli.py

```diff
@@ -483,6 +486,14 @@ def server(
 
 
 @cli.command(short_help="Permanently delete runs in the `deleted` lifecycle stage.")
+@click.option(
+    "--older-than",
+    default="0",
```

Comment:
The default value of `0` doesn't seem to conform to the proposed datetime string format of `#d#h#m#s`. Can we default to `None` instead? I think that this may also improve code readability.